.expeditor/integration_test.pipeline.yml

```diff
@@ -212,6 +213,7 @@ steps:
       SCENARIO: omnibus-tiered-upgrade
       PLATFORM: rhel-7
       ENABLE_IPV6: true
+      ENABLE_ADDON_PUSH_JOBS: false
```

Comment:
Thanks! We probably want to do the same thing in integration_test_full I think

Reply:
Yeah I was contemplating that, but I see the value in disabling it, since we have the issue to track that created. 👍